### DISCUSSION TITLE
log full of.... vethf3c3e95: Failed to get link config: No such device

### DISCUSSION THREAD

**Discussion by hxt21** [telekom-security/tpotce#1072] [Q&A]
I get 10 pr min of this : vethf3c3e95: Failed to get link config: No such device

--------------------------------------------------------------
CODE_FILE | src/udev/udev-builtin-net_setup_link.c
-- | --
CODE_FUNC | builtin_net_setup_link
CODE_LINE | 33
DEVICE | veth83dcd0f
PRIORITY | 3
SYSLOG_FACILITY | 3
SYSLOG_IDENTIFIER | systemd-udevd
TID | 128802
_BOOT_ID | 85d252c4add641a7ae47f365764ef604
_CAP_EFFECTIVE | 1f7fdffffff
_CMDLINE | /lib/systemd/systemd-udevd
_COMM | systemd-udevd
_EXE | /usr/bin/udevadm
_GID | 0
_HOSTNAME | rottenpiety
_MACHINE_ID | fe9557f2bfa44e92ae7f471665ef1041
_PID | 128802
_SELINUX_CONTEXT | unconfined
_SOURCE_REALTIME_TIMESTAMP | 1650031799122720
_SYSTEMD_CGROUP | /system.slice/systemd-udevd.service
_SYSTEMD_INVOCATION_ID | bcb0ed09be764dd8bace741e1b7b7bba
_SYSTEMD_SLICE | system.slice
_SYSTEMD_UNIT | systemd-udevd.service
_TRANSPORT | journal
_UID | 0
__CURSOR | s=bfa9ae271ea34a1d8a4ffd3d62171a34;i=20618;b=85d252c4add641a7ae47f365764ef604;m=123d1316c;t=5dcb1f7413b29;x=37c689c2477874c5
__MONOTONIC_TIMESTAMP | 4895879532
__REALTIME_TIMESTAMP | 1650031799122729
---------------------------------------------------------------------------

??????

**Comment by t3chn0m4g3** [comment]
[This](https://github.com/telekom-security/tpotce/blob/master/README.md#network-interface-fails) might help.

**Reply by hxt21** [reply]
thanks for the quick reply. I've had tpot 20.04 for a long time. I then 2 hours ago installed a fresh 22.04 tpot, and got an error that my net was not running, and read on your page how to fix it, and fixed this and that help. So tpot is up and running and I also get log on attacks from some honeypot, however my log is full of this error. Also has update tpot and it also works without problems, so.... Can't exactly see what the error is and how I fix it. Suricata won't work, though, I see. Thanks for your quick help. :)

**Reply by t3chn0m4g3** [reply]
During tests the exact error could be resolved with the description in the Readme. Since it is Debian under the hood you can try the Debian Wiki / forums for help or try the [Post Install Method](https://github.com/telekom-security/tpotce/blob/master/README.md#post-install).

**Reply by hxt21** [reply]
ok, thanks, I'll try to look at it. :)

**Reply by hxt21** [reply]
I can confirm this is because of docker, not Debian, no fix. :(
https://serverfault.com/questions/1074107/systemd-udevd-spams-systemd-udevd1523120-veth20996a5-failed-to-get-link-conf
https://forums.docker.com/t/docker-container-starting-without-network-not-adding-vethxxxxxxx-failed-to-get-link-config-no-such-device/119701/3

**Reply by t3chn0m4g3** [reply]
Thanks for the feedback, but strange.

**Reply by lpingree** [reply]
Am getting the same error.
